### ISSUE TITLE
Update ERC20 token contract address

### ISSUE THREAD

**Issue by Belance** [forkdelta/tokenbase#808]
Hello, Dear Decentralized Exchange ForkDelta! Hello, Dear @tradedbytony and @Kryptoniq  !

ERC20 team appreciated to be on Decentralized Exchange ForkDelta!

We appreciate a lot your hard work everyday and efforts on the ForkDelta project!

And we kindly asking you to update the information about our ERC20 token! 

**Since we opened and developing our project of our ERC20 token - blockchain platform Belance.  We are now happy to announce that ERC20 token contract address has been updated too.**

The following info on this text will guide you through the details.

Information about **OLD** Contract address;
**Old** Contract address:
0x26D5Bd2dfEDa983ECD6c39899e69DAE6431Dffbb

**Symbol: ERC**

Link to official explorer EtherScan: https://etherscan.io/address/0x26d5bd2dfeda983ecd6c39899e69dae6431dffbb

###  It would be great if you could change the current ERC20 to ERC OLD ( as symbol in old/current contract address ) . We are sorry for the inconvenience.

**NEW** Contract address: 

Contract Address:
0xc3761EB917CD790B30dAD99f6Cc5b4Ff93C4F9eA

**Name: ERC20**

**Symbol: ERC20**

Link to official explorer EtherScan: https://etherscan.io/address/0xc3761eb917cd790b30dad99f6cc5b4ff93c4f9ea#readContract

### **Contract Address:**
0xc3761EB917CD790B30dAD99f6Cc5b4Ff93C4F9eA

### **Issuer's official website:**
http://belance.io

### **About our ERC20 blockchain platform Belance:** 
http://belance.io/aboutbelance.html

### **Official contract address confirmation on the official website:** 
http://belance.io/erc20.html

### **A link to a third-party review or discussion of the token:**
https://bitcointalk.org/index.php?topic=2304875

### **Description:**

_What is the idea behind Belance?_

 _Belance is a large unique blockchain superior platform which combines many opportunities and connects everyone in the blockchain world. Whether you're building your new project or searching for something unique, Belance is the first platform where there are unlimited opportunities to connect and discuss with others regarding the cryptocurrency and blockchain industry. Also connect with professionals to help your own project by hiring a specialist to complete a specific task within your project. Promote and advertise your own skills and specialities to clients. Ask questions on any topic related to the blockchain or become a blockchain blogger. Belance is a one stop place for everything for everyone who is interested in or connected to the blockchain world._

**Warm regards,
ERC20 team**
belance.io

**Comment by Belance** [comment]
hi @Kryptoniq , sorry to disturb you. Can you please tell what is the approximately date of our update :) or what we can do or provide to make it more fast, thank you ! cant wait for any reply from you. Thanks!

**Comment by Belance** [comment]
hi, @freeatnet maybe you can also help with listing update ?) I know you are all busy guys, but we are waiting for this update more then 2 weeks ((( what we can do to make process faster ? thanks for any reply.

**Comment by Belance** [comment]
hi @Kryptoniq @freeatnet any news?

**Comment by Kryptoniq** [comment]
@Belance there are only 4 token holders on the new contract while the previous contract as over 7k token holders. Please do a token swap first  before we can consider your request. Thank you.

**Comment by Belance** [comment]
@Kryptoniq hello, first of all please note that we are not doing a swap and delisting, and we would not stop to support old (previous) erc20 - you also can check in official announcements on the website . Holders also will have opportunity to use old Erc20 on Belance platfrom as well as new erc20. 
so we are not swaping tokens. we are adding one more additional. so we will have 2 tokens like IDEX has - AURA(marketing token) and IDMEX ( main token ). so they also has 2  tokens - and it’s good example. so as you understand we also have now 2 tokens old ERC20(ERC) (matketing token) and new erc20(ERC20) ( main token). we have now 2 tokens by 3 reasons; first of all in old erc20 smart contract we made a mistake and symbol in smart contract is ERC instead of ERC20( you can check it and see by yourself ) . secondly we never did pre-sale, privat sale and ico for old erc20 - so thats why there is a lot of holders, because we did 2 free airdrops to all who wanted get coin for free (there was 6.000 + people ) , after we did 1 free airdrop for all holders ( 6.000+ because we made free aidrop for everyone), after we did 4 big bounties for reposts and finnally we burned the rest 86% of all old erc20 token ( as holders asked us, you can check it in ether scan first address). thridly - we just open our product/project - Belance blockchain platfrom a month ago so it’s understandable why old erc20 is marketing token because everyone get it for free and on the beginning when we did not have any opened project, but now we have our blockchain platfrom.  Also you cant us to tell to make a swap if we not planning to do a swap or delist old coin, because we just have an additional one. so logically is normal that new erc20 doesn’t have holders because  you did not list us yet and we not doing swaping and we not doing ico, privat sale, pre-sale for new erc20. so please kidly ask you to consider the request. very important that new erc20 will be used for safety transactions on our platfrom. once again about amount of the holders after you will consider our request amount of holders will increase because trades will start. aslo in our telegram chat ( 8.000+ members ) and channel chat (7.000+ members ) they already now about additional new erc20 token. and we all waiting for updates form you. also our old erc20 token previously ether delta listed by themselves) also our new erc20 token  already updated and listed on KUNA exchange, Token Store , Iquant and mercatox. also we are contacted to idex and ddex as well. so already everyone  updated us except you, ForkDelta. and when trades on all exchange will start you will have a lot of request to tech support because you are not updated us, and contract addrees will be different from all the rest exchanges. if you have questions please be free to ask. and  please kidly ask you to consider the request.

**Comment by Kryptoniq** [comment]
Yes, I have no right to tell you to make a swap if you didn't intend to but your issue is complicated. 
You have to file a new  listing request for your new token instead of telling us to   "update" the contract and then come back to say you don't want to delist the old token.

In that case, you/we have to follow due listing procedures.
If you want to update the old ERC20 Token address, please create a new issue for that.
If you want to list your new token Belance, please create a new issue for that also.

Thank you for your understanding.

**Comment by Kryptoniq** [comment]
Create a new issue for ERC20 contract update and then you can make this one solely for Belance listing.
